activation-config.yaml

```diff
@@ -16,6 +16,11 @@ googleCloud:
     # Require valid SSL certificate when communicating with Dynatrace Cluster / Active Gate
     # May be set to false in case communication through Active Gate is debugged, for non-production use cases
     requireValidCertificate: true
+    # if API calls should use default billing mechanism, `destination` if they should be billed per project

```

Comment:
I assumed it implicitly meant "current default value" at the beginning of the sentence. Something like `Require valid SSL certificate ...` above instead of `true to require valid SSL certificate ...` but yeah, it sounded confusing to me too